src/components/BalancesList.js

```diff
@@ -112,10 +112,11 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-function BalanceListItem({ publicKey }) {
+export function BalanceListItem({ publicKey, expandable }) {
```

Comment:
Re-used this component for the derivable accounts component, but without the ability to expand into details.